core/js/src/main/scala/spire/Platform.scala

```diff
@@ -0,0 +1,11 @@
+package spire
+
+import  scala.collection._
+
+class Platform {
+  type TrieMap[K, V] = mutable.Map[K, V]
```

Comment:
Because .... I had some other stuff in the Class, but not any more. Makes sense to clean it up